### PR TITLE
Add scope block support for backup schedule

### PR DESCRIPTION
**Pull request by shobha2626**
1. **What this PR does / why we need it**:  Add scope block support for backup schedule

2. **Which issue(s) this PR fixes**

        (optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged):

        Fixes #

3. **Additional information**

image

image

image

5. **Special notes for your reviewer**

### CODE REVIEW

File: examples/data-sources/backupschedule/data_source_backup_schedule.tf [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
   scope {
-    management_cluster_name = "MGMT_CLS_NAME"
-    provisioner_name        = "PROVISIONER_NAME"
-    cluster_name            = "CLS_NAME"
-    name                    = "TARGET_LOCATION_NAME"
+    cluster {
+      management_cluster_name = "MGMT_CLS_NAME"
+      provisioner_name        = "PROVISIONER_NAME"
+      cluster_name            = "CLS_NAME"
+      name                    = "TARGET_LOCATION_NAME"
```
Comment (eastaga) [inline examples/data-sources/backupschedule/data_source_backup_schedule.tf:7]:
```suggestion
      name                    = "BACKUP_SCHEDULE_NAME"
```

Comment (shobha2626) [reply examples/data-sources/backupschedule/data_source_backup_schedule.tf:7]: TARGET_LOCATION_NAME is as per the UI terminology, better to stick with the same

Comment (shobha2626) [reply examples/data-sources/backupschedule/data_source_backup_schedule.tf]: Done

**Review by eastaga — APPROVED** [review]
lgtm

**Review by vmw-vjn — APPROVED** [review]

**Review by rnarenpujari — APPROVED** [review]
LGTM. Thanks for taking care of this.

**Review by ankitsny — APPROVED** [review]

**Review by eastaga — APPROVED** [review]

**Review by ekoop-terasky — APPROVED** [review]
LGTM

**Pull request merged**